cpp/tests/utilities/timestamp_utilities.cuh

```diff
@@ -16,7 +16,7 @@
 
 #pragma once
 
-#include <simt/chrono>
+#include <cudf/utilities/chrono.hpp>
```

Comment:
Can the `timestamp_utilities` be subsumed by `fixed_width_column_wrapper`? Maybe perhaps be modified to provide iterators to feed into a `fixed_width_column_wrapper`?

Reply:
Yeah, it definitely can be.